internal/configurationmanagers/configfile/fs_watcher.go

```diff
@@ -7,11 +7,13 @@ import (
 	"github.com/fsnotify/fsnotify"
 )
 
+var logFatal = log.Fatal
```

Comment:
Why was this added?

Reply:
This is so we can mock the calls to logFatal without actually stopping execution.